libs/elastic-explorer/src/lib/modules/search/components/filters-form/filters-form.component.scss

```diff
@@ -0,0 +1,10 @@
+.my-fullScreen {
+  width: 100vw !important;
+  height: 100vh !important;
+  position: absolute;
+  z-index: 9999;
+}
+
+.my-editor-buttons {
+  margin-left: 0;
+}
```

Comment:
I don't think this does anything, there is no reason for that div to have a margin-left to begin with, no? 🤔

Reply:
I am surprised that only wrapping with div keeps it aligned to right 😕 (I thought the margin-left was also needed)